deploy/exec/delta-neutral-vault/config/init-position.ts

```diff
@@ -0,0 +1,219 @@
+  await provider.send("hardhat_impersonateAccount", [DEPLOYER_ADDRESS]);
+  const signer = provider.getSigner(DEPLOYER_ADDRESS);
+  const deployer = await SignerWithAddress.create(signer);
+  const tokenLists: any = config.Tokens;
+
+  for (let i = 0; i < initPositionInputs.length; i++) {
+    console.log("===================================================================================");
+    console.log(`>> Initializing position at ${initPositionInputs[i].symbol}`);
+    const stableVault = config.Vaults.find((v) => v.symbol === initPositionInputs[i].stableVaultSymbol);
+    const assetVault = config.Vaults.find((v) => v.symbol === initPositionInputs[i].assetVaultSymbol);
+    if (stableVault === undefined) {
+      throw `error: unable to find vault from ${initPositionInputs[i].stableVaultSymbol}`;
+    }
+    if (assetVault === undefined) {
+      throw `error: unable to find vault from ${initPositionInputs[i].assetVaultSymbol}`;
+    }
+
+    const stableTwoSidesStrat = stableVault.StrategyAddTwoSidesOptimal.Mdex;
+    const assetTwoSidesStrat = assetVault.StrategyAddTwoSidesOptimal.Mdex;
+
+    const deltaNeutralVaultConfig = DeltaNeutralVaultConfig__factory.connect(
+      initPositionInputs[i].deltaNeutralVaultConfig,
+      deployer
+    );
+
+    console.log(">> Setting leverage level at DeltaNeutralVaultConfig to be 3x");
+    await deltaNeutralVaultConfig.setLeverageLevel(3);
+    console.log("✅ Done");
+
+    const stableToken = tokenLists[initPositionInputs[i].stableSymbol];
+    const assetToken = tokenLists[initPositionInputs[i].assetSymbol];
+    const stableTokenAsDeployer = BEP20__factory.connect(stableToken, deployer);
+    const assetTokenAsDeployer = BEP20__factory.connect(assetToken, deployer);
+
+    await stableTokenAsDeployer.approve(initPositionInputs[i].deltaVaultAddress, ethers.constants.MaxUint256);
+    await assetTokenAsDeployer.approve(initPositionInputs[i].deltaVaultAddress, ethers.constants.MaxUint256);
+
+    const deltaNeutralOracle = DeltaNeutralOracle__factory.connect(DELTA_NEUTRAL_ORACLE_ADDR, deployer);
+    const [stablePrice] = await deltaNeutralOracle.getTokenPrice(stableToken);
+    const [assetPrice] = await deltaNeutralOracle.getTokenPrice(assetToken);
+
+    console.log(`stablePrice: ${stablePrice}`);
+    console.log(`assetPrice: ${assetPrice}`);
+
+    // open position1
+    console.log(">> Preparing input for position 1 (StableVaults)");
+    const assetAmount = ethers.utils.parseEther(initPositionInputs[i].assetAmount);
+    console.log(`>> assetAmount: ${assetAmount}`);
+    const principalStableAmount = assetAmount.mul(assetPrice).div(stablePrice);
+    console.log(`>> principalStableAmount: ${principalStableAmount}`);
+
+    const principalStablePosition = principalStableAmount.mul(BigNumber.from("1")).div(BigNumber.from("4"));
+    console.log(`>> principalStablePosition: ${principalStablePosition}`);
+
+    const farmingTokenStablePosition = ethers.utils.parseEther("1");
```

Comment:
To simplify calculation, don't deal with asset token.
```suggestion
    const farmingTokenStablePosition =  ethers.utils.parseEther("0");
```